.changeset/add_bus_routes_to_openapi_spec.md

```diff
@@ -0,0 +1,5 @@
+---
+default: major
+---
+
+# Add bus routes to openapi spec
```

Comment:
I'd update this changeset. Extending the spec isn't really breaking anything so I'd make it a minor version. Also I'd list the sub routes that you add because otherwise it seems like this PR adds all bus routes.